thirdparty/simpletest/page.php

```diff
@@ -90,7 +90,7 @@ function _createInputTag($attributes) {
                 'hidden' => 'SimpleTextTag',
                 'password' => 'SimpleTextTag',
                 'file' => 'SimpleUploadTag');
-        if (array_key_exists($type, $map)) {
+        if (array_key_exists($type, $map) || $type = 'text') {
```

Comment:
Um, `text` is IN the $map. The second condition will never be true when the first is false.

Reply:
Oh right, assignment via condition.... this is why it's a bad practice. Can you rewrite as a pure assignment to $tag_class instead of a condition?